config/default/common/config/metadata/layers/amsr2/AMSRU2_Cloud_Liquid_Water_Day.md

```diff
@@ -4,4 +4,4 @@ The AMSR2 instrument is a conically scanning passive microwave radiometer. This
 
 The imagery resolution is 2 km and sensor resolution is 6.25 km. The temporal resolution is daily.
 
-References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01)
+References: AU_OCEAN_NRT [doi:10.5067/AMSRU/AU_OCEAN_NRT_R01](https://doi.org/10.5067/AMSRU/AU_OCEAN_NRT_R01); AU_OCEAN [doi:10.5067/9YQRFKKEPUP5](https://doi.org/10.5067/9YQRFKKEPUP5)
```

Comment:
This DOI link is NOT FOUND.

Reply:
Looks like they gave me the wrong link and was a number off, I've fixed them!